lib/ParkingSpace.php

```diff
@@ -19,12 +21,25 @@ public function getAllSpaces(){
         return $results;
     }
 
-    // GET ALL OPEN SPACES
-    public function getAllOpenSpaces($lotID){
+    // GET ALL DISTINCT LOT_IDS
+
+    public function getAllDistinctLotIDs()
+    {
         $this->db->query("
-            SELECT COUNT(Is_Occupied) AS NumberOfOpenSpots
+            SELECT DISTINCT Lot_ID
+            FROM parking_space;
```

Comment:
fix case sensitivity, please.

Reply:
I don't even have this method anymore in my ParkingSpace.php